wordpress-mobile/WordPress-iOS
# Update Aztec to version 1.5.0 beta 2

This PR updates Aztec to version 1.5.0 beta 2

This version brings fixes for the following:
 - video shortcodes are only used when inserted by the app or the original content was using them.
 - add the possibility of having videos with multiple sources in the content.

To test:
 - Open the app
 - On the Aztec editor open post with video content using shortcodes `[video]` or `[wpvideo]` and with the standard HTML video element `<video>` (WP.com and WP.org)
 - Check that all video shows correctly
 - Insert videos using the app and see that video insertion is done correctly
 - Create a post using the HTML with a video element with multiple sources check that is displayed correctly when switching to visual.

Update release notes:
